GSDemo/app/build.gradle

```
@@ -1,16 +1,14 @@
 apply plugin: 'com.android.application'
 
-def boostDirPath = "/home/konglobemeralt/Documents/gitz/Boost-for-Android/build/out/arm64-v8a"
+
+def boostDirPath = "/home/globee/AndroidStudioProjects/Boost-for-Android/build/out/arm64-v8a"
```

```suggestion
def boostDirPath = //Add your boost dir
```
We are guilty of hardcoding this as well, so I am not complaining that you did this.

Boost for android should probably be added as a submodule to the project to allow the project dir to contain the boost files and allow for a relative path to the relevant files. I will look into adding this in the future.